debian/common_context/entrypoint.sh

```diff
@@ -9,27 +9,44 @@
 #   Felix Schindler (2017)
 #   Rene Milk       (2017)
 
-USERNAME_=${LOCAL_USER:-user}
-UID_=${LOCAL_UID:-1000}
+USERNAME_=${LOCAL_USER:-root}
+UID_=${LOCAL_UID:-0}
 GID_=${LOCAL_GID:-$UID_}
 
-groupadd -g $GID_ $USERNAME_ &> /dev/null
-if [ -e /home/$USERNAME_ ] ; then
-  useradd -d /home/$USERNAME_ -g $GID_ -s /bin/bash -u $UID_ $USERNAME_
-else
-  useradd -md /home/$USERNAME_ -g $GID_ -s /bin/bash -u $UID_ $USERNAME_
-fi
+export LANG=en_US.UTF-8
```

Comment:
This should already be baked into the image. If would be better not to repeat it then.

Reply:
agreed. similarly the hosts definition, I guess?

Reply:
Actually, probably no (see above)?

Reply:
k, will prepare another pr. will also try to adapt the docker_run accordingly.